node/full_node_integration_test.go

```diff
@@ -169,146 +172,140 @@ func TestLazyAggregator(t *testing.T) {
 
 	require.NoError(err)
 
-	// delay to ensure it waits for block 1 to be built
-	time.Sleep(1 * time.Second)
+	require.NoError(waitForFirstBlock(node.(*FullNode)))
 
 	client := node.GetClient()
 
 	_, err = client.BroadcastTxCommit(context.Background(), []byte{0, 0, 0, 1})
 	assert.NoError(err)
-	time.Sleep(2 * time.Second)
-	assert.Equal(node.(*FullNode).Store.Height(), uint64(2))
+	require.NoError(testutils.Retry(300, 100*time.Millisecond, func() error {
```

Comment:
can't these three calls use the helper `waitForAtLeastNBlocks`

Reply:
yep